doc/Makefile

```diff
@@ -29,6 +30,9 @@ freqplot-siso_bode-default.png: ../control/tests/freqplot_test.py
 rlocus-siso_ctime-default.png: ../control/tests/rlocus_test.py
 	PYTHONPATH=.. python $<
 
+phaseplot-dampedosc-default.png: ../control/tests/phaseplot_test.py
+	PYTHONPATH=.. python $<
```

Comment:
Should the other images generated by phaseplot_test.py (`phaseplot-oscillator-helpers.png`, `phaseplot-invpend-meshgrid.png`) be included here? It does not seem important because the PNG files are being committed, but I note this here in case you wanted to.

Reply:
Agree that they could be listed, but I didn't list them all out since triggering on the one file will update all files => if the test generation script is updated, all PNG files are updated.

Reply:
Good point! I will complete review tomorrow.